pkg/codegen/schema/schema.go

```diff
@@ -245,6 +247,8 @@ type Alias struct {
 
 // Resource describes a Pulumi resource.
 type Resource struct {
+	// Package is the package that defines the resource.
+	Package *Package
```

Comment:
Probably also want to add this to `Function`.

Reply:
Done